apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/model/DexTradeEntry.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.exchange.model;
+
+import com.apollocurrency.aplwallet.api.dto.DexTradeInfoDto;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Data class for interaction with trade table in the database
+ * @author Serhiy Lymar
+ */
+
+@Getter @Setter
+public class DexTradeEntry {
+
+    private long dbId;
+    private long transactionID;
+    private long senderOfferID; 
+    private long receiverOfferID;
+    private byte senderOfferType; 
+    private byte senderOfferCurrency; 
+    private long senderOfferAmount;
+    private byte pairCurrency;
+    private BigDecimal pairRate;
+    private Integer finishTime;
+    private Integer height; 
+
+    public DexTradeInfoDto toDto(){       
```

Comment:
I guess it's better to move the DTO conversion into separated service that implements the core/rest/converter/Converter interface.  The model class hasn't to contain any translating logic.

Reply:
done